.gitlab-ci.yml

```diff
@@ -363,3 +363,11 @@ syntax-check:
   script:
     - ansible-playbook -i inventory/local-tests.cfg -u root -e ansible_ssh_user=root  -b --become-user=root cluster.yml -vvv  --syntax-check
   except: ['triggers']
+
+tox-inventory-builder:
+  stage: unit-tests
+  <<: *job
+  script:
+    - cd contrib/inventory-builder && tox
+  when: manual
```

Comment:
Yes that's intentional. Unless you can set a trigger that runs it only when contrib/inventory-builder is modified